js/cd-print.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * @file
+ * Print helper.
+ */
+
+((Drupal) => {
+  'use strict';
+
+  Drupal.behaviors.cdPrint = {
+    attach: function (context, settings) {
+
+      // Remove lazy loading before printing.
+      window.addEventListener('beforeprint', (event) => {
+        document.querySelectorAll('[loading="lazy"]').forEach(element => {
+          element.removeAttr('loading');
```

Comment:
```suggestion
          element.removeAttribute('loading');
```

`removeAttr` is jQuery, please use [removeAttribute](https://developer.mozilla.org/en-US/docs/Web/API/Element/removeAttribute).

Reply:
ouch. sorry.